package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "description": "Lumeer Web UI",
   "main": "index.js",
+  "config": {
+    "ENV_VAR": "echo $bla"
```

Comment:
Do we need this or did you just use it for testing?

Reply:
It was for testing, I'll remove it.